client/deposits/list/index.js

```diff
@@ -129,17 +127,24 @@ export const DepositsList = () => {
 		return columns.map( ( { key } ) => data[ key ] || { display: null } );
 	} );
 
-	const summary = [
-		{
-			label: __( 'deposits', 'woocommerce-payments' ),
-			value: `${ depositsSummary.count }`,
-		},
-	];
-
 	const isCurrencyFiltered = 'string' === typeof getQuery().store_currency_is;
-	if ( ! isSummaryLoading ) {
-		const isSingleCurrency =
-			2 > ( depositsSummary.store_currencies || [] ).length;
+	const isSingleCurrency =
+		2 > ( depositsSummary.store_currencies || [] ).length;
+
+	let summary;
+	const isDespositsSummaryDataLoaded =
+		depositsSummary.count !== undefined &&
+		depositsSummary.total !== undefined;
```

Comment:
There are more attributes to test if the data has been loaded but I believe checking these two should suffice, for now, let me know if more properties need to be checked.